lib/fprotect/Kconfig

```diff
@@ -50,3 +50,10 @@ config FPROTECT_BLOCK_SIZE
 	default $(dt_node_int_prop_hex,$(DT_CHOSEN_ZEPHYR_FLASH),erase-block-size)
 	help
 	  Block size of fprotect
+
+config FPROTECT_RESERVE_HUK_ACL_SLOT
+	bool "Reserve the last ACL slot for HUK"
+	depends on SOC_NRF52840
+	help
+		This will reserve the last ACL slot for the hardware unique key (HUK).
```

Comment:
Since this is nRF52840, should the ACL slot reserved be named (i.e. the actual number)?

Reply:
I don't see any problem with that I think. I can hardcode the last ACL slot.